client/src/components/Landing.css

```diff
@@ -0,0 +1,46 @@
+
+#background {
+    
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+} 
```

Comment:
What's the point of this background class? I removed the CSS and it doesn't make any differences. Its child <img> already takes up 100% width and 1000px?